config/jest.ci.js

```diff
@@ -0,0 +1,28 @@
+const path = require('path');
+
+module.exports = {
+  moduleFileExtensions: ['ts', 'js'],
+  testEnvironment: 'node',
+  maxWorkers: '50%',
+  rootDir: process.cwd(),
+  // The UI and server packages should follow its jest config
```

Comment:
not related to this project

Reply:
Please check other config, such as maxWorkers

Reply:
@Yuyz0112 
Sorry about the mistake, I had removed some of the irrelevant config but still left some here.
I have updated the config for ci and modify some babel config for unit test, `maxWorkers:'50%'` was kept due to it is the best practice for jest to run in environments with variable CPUs available.
PTAL